feathr_project/feathr/definition/settings.py

```diff
@@ -3,29 +3,66 @@
 from loguru import logger
 from feathr.definition.feathrconfig import HoconConvertible
 
+class ConflictsAutoCorrection():
```

Comment:
Oh interesting. This is updated code from the official feathr repo that hasn't been merged into this repo yet right?